pkg/v6/configmap/desired.go

```diff
@@ -231,3 +239,15 @@ func newConfigMapLabels(configMapValues ConfigMapValues, appName, projectName st
 		label.ServiceType:  label.ServiceTypeManaged,
 	}
 }
+
+// setIngressControllerTempReplicas sets the temp replicas to 50% of the worker
+// count to ensure all pods can be scheduled.
+func setIngressControllerTempReplicas(workerCount int) (int, error) {
+	if workerCount == 0 {
+		return 0, microerror.Maskf(invalidExecutionError, "worker count must not be 0")
+	}
+
+	tempReplicas := float64(workerCount) * float64(0.5)
+
+	return int(math.Round(tempReplicas)), nil
```

Comment:
Rounding up is so on single node clusters we run 1 temp replica. On 3 node clusters we run 2 temp replicas.